calculator_v5.rb

```diff
@@ -0,0 +1,93 @@
+# Goal is to create a program that takes two numbers and performs mathematical operations on them, then gives the result. Shooting for Gold level here.
+
+# outputs welcome string and user prompt
+puts "Welcome to Mathtropolis! Let's do some math!"
+puts "What type of math would you like to do?"
+
+# assigns user input to the variable "operation"
+operation = gets.chomp.downcase
+
+# defines a method called do_math that takes 3 arguments corresponding to
+# operation and inputted numbers below
+def do_math(operation, num1, num2)
+
+  # if operation is equal to values indicating addition
+  if operation == "+" || operation == "add" || operation == "addition" || operation == "adding"
```

Comment:
I wanted to revise that before making the pull request but ran out of time. Today's commit includes it! Yay!